Cargo.toml

```diff
@@ -4,3 +4,5 @@ members = [
     "trin-state",
     "trin-history",
 ]
+
+default-members = ["trin"]
```

Comment:
Does this change the behavior of any Cargo commands?

Reply:
If we don't have this, running `cargo run` won't default to running `trin/src/main.rs`, it will instead say: 
```error: cargo run could not determine which binary to run. Use the --bin option to specify a binary, or the default-run manifest key. available binaries: trin, trin-history, trin-state```.

I don't think that running `trin` by itself will be very useful for a while, so maybe it wouldn't be bad to not have this and force a choice for now. 

Also, this makes it so that we have to run `cargo run -p trin-state` instead of `cargo run --bin trin-state` the latter of which seems way more common in Rust.

So maybe we should get rid of this. I'll keep it in for now and we'll see if we hate it.